### DISCUSSION TITLE
SenseCAP S2110 Sensor Builder | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1698] [Q&A]
# SenseCAP S2110 Sensor Builder | Seeed Studio Wiki

Overview of SenseCAP S2110 Sensor Builder

https://wiki.seeedstudio.com/Build-LoRaWAN-Sensors-SenseCAP-XIAO-Controller-Data-Logger/

**Comment by eerke2021** [comment]
As per the Hazard livestream I am thinking of adding a sensitive vibration sensor piezoelectric to this in order to detect earthquakes and/or a radiation detector to this but get lost in circles whenever I go to GitHub Contributor. Guessing this just C on the RP2040 programmed using the Arduino software and the code to adjust is above? I don’t think there is a grove earthquake or nuclear radiation sensor. Then you could go earthquake_sensor to Sensor Builder to Data Logger to LoRa to M2 Multiplatform running chirpstack local mode. Assume the data logger TTN code would also need adjusting but don’t know yet the RS485 protocol. That would be an interesting, if pointless as I don’t live in an earthquake zone and too near an obvious nuclear target to survive, project. Let me know if you are already developing a Grove earthquake or Radiation sensor and I will not reinvent the wheel.
